src/pages/Thanks/ThanksPage.vue

```diff
@@ -294,6 +298,7 @@ export default {
 
 			const achievements = userAchievements?.userAchievementProgress?.tieredLendingAchievements ?? [];
 			this.achievementsCompleted = this.allAchievementsCompleted(achievements);
+			this.thanksPageGoalsEntrypointEnable = readBoolSetting(data, `general.${THANK_YOU_PAGE_GOALS_ENABLE_KEY}.value`); // eslint-disable-line max-len
```

Comment:
Would be worth a `?? false` here, since `readBoolSetting` returns `null` as fallback?

Reply:
Nice first formalism for this case!